sdk/rust/src/processor/handler.rs

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright 2017 Bitwise IO, Inc.
```

Comment:
2018?

Reply:
2017 is accurate, it started late last year.